### PR TITLE
Sidecar tasks are now Pod tasks

### PR DESCRIPTION
**Pull request by katrogan**

### CODE REVIEW

File: flytekit/taskplugins/pod/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from .task import Pod
+
+__all__ = [Pod]
```
Comment (kumare3) [inline flytekit/taskplugins/pod/__init__.py:3]: so the `__all__`` should be a list of string names of the modules you want to expose. 
I think you can just skip it, refer to my new PR

**Review by kumare3 — APPROVED** [review]
lgtm, we can wait for @jeevb to comment?

**Comment by katrogan** [comment]
thanks @kumare3 also posted in flytekit slack and will wait until next week to see if anyone has strong opinions

**Review by jeevb — APPROVED** [review]
LGTM! To clarify, does this seamlessly work with dynamic tasks?

**Comment by katrogan** [comment]
@jeevb thanks for reviewing! we haven't yet ported over dynamic sidecar tasks (dynamic pod tasks?) to the new flytekit but that is in the pipeline

**Pull request merged**